service/FactomService.js

```diff
@@ -43,14 +29,22 @@ exports.deactivate = function (body) {
 exports.register = function (body) {
     return new Promise(function (resolve, reject) {
         const { jobId: entryHash, options } = body;
+        const network = options.network || Networks.MAIN;
+        if (network !== Networks.MAIN || network !== Networks.TEST) {
+            reject(new InvalidNetworkParameterError(
+                "Unsupported network specified! Must be test or main"));
```

Comment:
Provide the name of the supplied network in the error

Reply:
Updated.